test/kpod_kill.bats

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bats
+
+load helpers
+
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
+function teardown() {
+    cleanup_test
+}
+
+@test "stop a bogus container" {
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} kill foobar
+    echo "$output"
+    [ "$status" -eq 1 ]
+}
+
+@test "kill a running container by id" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run crioctl ctr start --id "$ctr_id"
+    echo "$output"
+    id="$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} kill "$id"
+    [ "$status" -eq 0 ]
+    cleanup_pods
+    stop_crio
+}
+
+@test "kill a running container by id with TERM" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run crioctl ctr start --id "$ctr_id"
+    echo "$output"
+    id="$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} kill -s TERM "$id"
+    [ "$status" -eq 0 ]
+    cleanup_pods
+    stop_crio
+}
+
+@test "kill a running container by name" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run crioctl ctr start --id "$ctr_id"
+    echo "$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} kill "k8s_podsandbox1-redis_podsandbox1_redhat.test.crio_redhat-test-crio_0"
+    [ "$status" -eq 0 ]
+    cleanup_pods
+    stop_crio
+}
+
+@test "kill a running container by name with bogus signal" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run crioctl ctr start --id "$ctr_id"
+    echo "$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} kill -s foobar "k8s_podsandbox1-redis_podsandbox1_redhat.test.crio_redhat-test-crio_0"
+    [ "$status" -eq 1 ]
```

Comment:
Will the exit code always be 1 for the error? another option is to use `["$status" -ne 0]`.

Reply:
It would have been in the previous implementation where we explicitly exited with a 1.  But now that is changed and ne 0 probably makes more sense.